pkg/cvo/availableupdates.go

```diff
@@ -40,9 +40,17 @@ func (optr *Operator) syncAvailableUpdates(ctx context.Context, config *configv1
 
 	// updates are only checked at most once per minimumUpdateCheckInterval or if the generation changes
 	u := optr.getAvailableUpdates()
-	if u != nil && u.Upstream == upstream && u.Channel == channel && u.RecentlyChanged(optr.minimumUpdateCheckInterval) {
-		klog.V(4).Infof("Available updates were recently retrieved, will try later.")
+	if u == nil {
+		klog.V(4).Info("First attempt to retrieve available updates")
```

Comment:
Why not add upstream url for logging?

Reply:
We log that later, when we actually make the request:

```console
$ git --no-pager grep 'to request updates from' pkg/cincinnati
pkg/cincinnati/cincinnati.go:                   klog.V(5).Infof("Using a root CA pool with 0 root CA subjects to request updates from %s", uri)
pkg/cincinnati/cincinnati.go:                   klog.V(5).Infof("Using a root CA pool with %n root CA subjects to request updates from %s", len(c.transport.TLSClientConfig.RootCAs.Subjects()), uri)
pkg/cincinnati/cincinnati.go:                   klog.V(5).Infof("Using proxy %s to request updates from %s", proxy.Host, uri)
```

In between this line and that, we can still bail out for reasons like "channel is empty"